### ISSUE TITLE
Does not work

### ISSUE THREAD

**Issue by sunilsankar** [chrishubert/whatsapp-api#223]
Currently this api is not working. It is unable to get the session established

**Comment by avoylenko** [comment]
It would be more useful if you can provide more details about the issue.

**Comment by sunilsankar** [comment]
After qrcode scan session is not getting connected

**Comment by avoylenko** [comment]
The QR code is being refreshed every one minute, so might scan an invalid code.

**Comment by sunilsankar** [comment]
Qr code gets scanned but session does not get connected

**Comment by avoylenko** [comment]
Any errors in the service logs? Do you run the latest docker image?

**Comment by sunilsankar** [comment]
There is no error . And I am using the latest image.

**Comment by fichaepi** [comment]
we noticed this problem today.  messages not going through. 
upon restart we had this error logged:
```
 Initialize error: Evaluation failed: TypeError: Cannot read properties of undefined (reading 'push')
at fillModuleArray (eval at <anonymous> (:2:5), <anonymous>:6:74)
at moduleRaid (eval at <anonymous> (:2:5), <anonymous>:15:3)
at __puppeteer_evaluation_script__:4:17
```

we fixed it by following these directions 
https://github.com/pedroslopez/whatsapp-web.js/issues/1786#issuecomment-2052231033

our take:
- updated whatsapp-api to latest package (as of  june 4th)
- updated .env like so:

WEB_VERSION="2.2412.54"
WEB_VERSION_CACHE_TYPE="remote"

- restart pm2
- restart session  ( /session/restart )
- rescanned qr code

it finally worked again!

hope it helps.
